personalize.api.php

```diff
@@ -56,15 +56,14 @@ function hook_personalize_option_set_type() {
 }
 
 /**
- * An agent is about to be created or updated.
- *
- * This hook is invoked before the agent is saved to the database so
- * modules can alter the information that is saved.
+ * Allows the alteration of an agent before it is saved.
  *
  * @param stdClass $agent
  *   A stdClass object representing the agent being saved.
+ * @param array $prior_values
+ *   The unalterable prior data for the agent before this save.
  */
-function hook_personalize_agent_presave($agent) {
+function hook_personalize_agent_alter(&$agent, $prior_values) {
```

Comment:
Objects are always passed by reference. That is weird that we're no longer invoking hook_personalize_agent_presave() especially seeing as there's an implementation of it in acquia_lift :P Anyway, _alter just feels wrong for this as it's usually something you'd use for altering something that's about to be used, not something that's about to be saved to the db.